spec/writers/sparql_writer_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'traject'
+require 'rialto/etl/writers/sparql_writer'
+
+RSpec.describe Rialto::Etl::Writers::SparqlWriter do
+  subject(:writer) do
+    described_class.new(
+      'sparql_writer.update_url': 'http://127.0.0.1:9999/sparql',
+      'sparql_writer.thread_pool': 0,
+      'sparql_writer.batch_size': batch_size,
+      'logger': logger
+    )
+  end
+
+  let(:logger) { instance_double('Yell::Logger') }
+
+  let(:batch_size) { 1 }
+
+  before do
+    allow(logger).to receive(:warn)
+    allow(logger).to receive(:info)
+    allow(logger).to receive(:debug)
+  end
+
+  describe '#put' do
+    let(:batch) do
+      Traject::Indexer::Context.new(source_record: 'SPARQL INSERT STATEMENT')
+    end
+
+    describe 'when returns 200' do
+      before do
+        stub_request(:post, 'http://127.0.0.1:9999/sparql')
+          .with(
+            body: 'SPARQL INSERT STATEMENT',
+            headers: {
+              'Content-Type' => 'application/sparql-update; charset=utf-8'
+            }
+          )
+          .to_return(status: 200, body: '', headers: {})
+      end
+
+      it 'posts the statement' do
+        writer.put(batch)
```

Comment:
Do you need an expectation of some sort here? I'm not sure what's being tested here.

Reply:
Like, expect not to raise error, expect not to log a warning, etc.